app/assets/stylesheets/application.css.scss

```diff
@@ -202,3 +202,9 @@ select {
 #container .mdl-grid {
   padding: 8px 0;
 }
+
+#container .mdl-layout-title{
```

Comment:
Avoid using id selectors<br>Merge rule `#container .mdl-layout-title` with rule on line 128<br>Opening curly brace `{` should be preceded by one space